include/unifex/stop_if_requested.hpp

```diff
@@ -52,6 +52,13 @@ struct _fn {
     // Provide an awaiter interface in addition to the sender interface
     // because as an awaiter we can take advantage of symmetric transfer
     // to save stack space:
+    //
+    // TODO: the scheduler affinity changes have mooted this because we check
+    //       for sender-ness before awaitable-ness; two possible solutions:
+    //         - check for awaitable-ness first
+    //         - customize await_transform here to do something smarter than the
+    //           default implementation; perhaps just return *this, but perhaps
+    //           we could remove the branch in await_suspend
```

Comment:
This is not actually mooted by this work.  We _do_ now check for sender-ness before awaitable-ness in `task<>::promise_type::await_transform()`, but, for general senders like this one, we delegate to `unifex::await_transform()`, which checks for awaitable-ness _first_.